### DISCUSSION TITLE
managed/bsp/float

### DISCUSSION THREAD

**Discussion by 24unix** [asmvik/yabai#779] [Q&A] [answered]
Is bsp identical to managed or are there differences?

**Comment by asmvik** [comment, marked as answer]
Managed means it is either in a bsp or stacking space, and is not toggled floating.

Unmanaged windows are either floating or in a float-space.

Sent from my iPhone

On 27 Dec 2020, at 18:25, Micha <notifications@github.com> wrote:

﻿

Is bsp identical to managed or are there differences?

**Comment by 24unix** [comment]
Thanks. So I need a Picker managed, bsp and float, right?

Currently it looks like so:

<img width="945" alt="image" src="https://user-images.githubusercontent.com/36622631/103281938-8b116700-49d4-11eb-9981-24386025b6f6.png">

**Comment by asmvik** [comment]
Well, bsp, stack and float may make more sense.

Sent from my iPhone

On 29 Dec 2020, at 12:52, Micha <notifications@github.com> wrote:

﻿

Thanks. So I need a Picker managed, bsp and float, right?

Currently it looks like so:

[image]<https://user-images.githubusercontent.com/36622631/103281938-8b116700-49d4-11eb-9981-24386025b6f6.png>

**Comment by 24unix** [comment]
OK, thanks, I will adapt it.
